hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/RolloutManagementTest.java

```diff
@@ -947,41 +995,113 @@ public void findRolloutGroupTargetsWithRsqlParam() {
 
     }
 
-    @Test
-    @Description("Verify the creation and the start of a rollout in asynchronous mode.")
-    public void createAndStartRolloutInAsync() throws Exception {
+    @Test(expected = RolloutVerificationException.class)
+    @Description("Verify the creation of a Rollout without targets throws an Exception.")
+    public void createRolloutNotMatchingTargets() {
```

Comment:
It is better to explain, why we should not use the `expected = RolloutVerificationException.class`.

First reason is the code covergage is not working with this annotation.
And here are some other examples: https://pettermahlen.com/2012/02/07/dont-use-testexpected/